tools/compiletest/src/runtest.rs

```diff
@@ -317,10 +317,12 @@ impl<'test> TestCx<'test> {
         let function_name = self.testpaths.file.file_stem().unwrap().to_str().unwrap();
         cargo
             .arg("kani")
-            .args(["--function", function_name])
             .arg("--target-dir")
             .arg(self.output_base_dir().join("target"))
             .current_dir(&parent_dir);
+        if "expected" != self.testpaths.file.file_name().unwrap() {
+            cargo.args(["--function", function_name]);
```

Comment:
Do we have a jira to change these to `--harness`?

Reply:
Changed them to harness now.